src/version.h

```diff
@@ -1,4 +1,5 @@
 #define SERVER_NAME "valkey"
+#define SERVER_TITLE "Valkey"
```

Comment:
Let's add a comment to clarify the difference between these two macros.

```suggestion
/* These macros are used where the server name is printed in logs and replies.
 * Note the difference in the first letter "V" vs "v". SERVER_TITLE is used in
 * readable text like log messages and SERVER_NAME is used in INFO fields and
 * similar. */
#define SERVER_NAME "valkey"
#define SERVER_TITLE "Valkey"
```

Reply:
Thanks